ml_tools/eolearn/tests/test_train_split.py

```diff
@@ -96,60 +103,36 @@ def test_train_split() -> None:
         ignore_values=[2],
     )
 
-    patch = split_task(patch, seed=542)
-
-    assert set(np.unique(patch[(FeatureType.MASK_TIMELESS, "BINS")])) <= expected_unique
-    assert np.all(patch[(FeatureType.MASK_TIMELESS, "BINS")][data == 2] == 0)
+    eopatch1 = split_task(eopatch1)
```

Comment:
consider using a different name, such as `result_eopatch = split_task(eopatch1)`. Adjust elsewhere as well, just in case

Reply:
done